mplc/doc/documentation.md

```diff
@@ -536,16 +551,9 @@ The data labels can take whatever shape you need, with only one condition. The l
 
 The `Dataset` object must provide separate train and test sets (referred to as global train set and global test set).
 
-The global train set is then further split into a global train set and a global validation set, by the function `train_val_split_global()`. Please note that if this function is not overwritten, the SKLearn's `train_test_split()` function will be called by default, and `10%` of the training set will be use as validation set. 
-
-In the multi-partner learning computations, the global validation set is used for early stopping and the global test set is used for performance evaluation.
-
-The global train set is then split amongst partners (according to the scenario configuration) to populate the partner's local datasets.
-
-For each partner, the local dataset will be split into separated train, validation and test sets, using the `train_test_split_local()` and `train_val_split_local()` functions.
-These are not mandatory, by default the local dataset will not be split. 
+The global train set is then further split into a global train set and a global validation set, in a stratified way. Please note that you can specified the proportion of the train set used for validation via the `val_proportion` keyword argument. Default is `0.10`.
```

Comment:
I would remove `in a stratified way` as we don't use any stratification strategy for the global and train/val set ?

Reply:
We do now, as some scenario have unbalanced label representation

Reply:
same, see 288

Reply:
Ah nice !!!